content/guides/cli/scripts/extract-metadata.md

```diff
@@ -0,0 +1,133 @@
+---
+rank: 7
+related_endpoints: []
+related_guides:
+  - authentication/oauth2
+  - cli/quick-start/powershell-script-templates
+related_pages:
+  - sdks-and-tools
+required_guides:
+  - authentication/oauth2/oauth2-setup
+related_resources: []
+
+---
+# Extract metadata
+
+This script extracts metadata details for all the
+files and folders in any Box folder and
+saves the result in a CSV spreadsheet for
+each metadata template.
+
+## Prerequisites
+
+### Windows
+
+Install the latest version of [dotnet core](https://dotnet.microsoft.com/download).
+
+### MacOS & Linux
+
+Install [PowerShell][pwsh]. Run the `pwsh` command to test the installation.
+
+   ```bash
+    pwsh 
+   ```
+
+Depending on the directory you are
+running the command in, the output may differ.
+For example:
+
+   ```bash
+   PowerShell 7.2.5
+   Copyright (c) Microsoft Corporation.
+
+   https://aka.ms/powershell
+   Type 'help' to get help.
+     
+   PS /Users/user/repos/boxcli/examples> 
+   ```
+
+   <Message>
+      If you encounter issues make sure you installed both 
+      [dotnet core](https://dotnet.microsoft.com/download) and 
+      [PowerShell][pwsh].
+   </Message>
+
+### Box application
+
+To use the script, you will need a Box application
+with OAuth 2.0 authentication. If you don't have one,
+go to your [Developer Console][console], and follow the 
+[Setup with OAuth 2.0][auth] guide.
+
+## Configure the script 
+
+1. Clone the `boxcli` GitHub repository 
+   or download the files from [`examples`][examples] directory.
+
+   ```bash
+     git clone https://github.com/box/boxcli.git
+   ```
+
+2. Specify the `folderID` of the folder you want the script to scan
+   for metadata.
+
+   ```bash
+   [string]$FolderID = "",
+   ```
+
+3. Specify the `userID` if you want to run the script on behalf of a different  
+   user.
+
+   ```bash
+   [string]$UserID = "",
+   ```
+
+   <Message>
+   If you don't specify parameters explicitly, the script
+   will prompt you to enter them.
+   </Message>
```

Comment:
I would mention that user can either pass parameters as flags (e.g. `./Metadata-extraction.ps1 -folderId 123456789 -userId 123456789`) OR the script will prompt user for params